backend/server.py

```diff
@@ -37,19 +39,27 @@ def api(app_name, api_call):
 Database State:
 {db[app_name]["state"]}
 
-Output the API response prefixed with 'API response:'. Then output the new database state as json, prefixed with 'New Database State:'. If the API call is only requesting data, then don't change the database state, but base your 'API Response' off what's in the database.
+Output the API response as json prefixed with '!API response!:'. Then output the new database state as json, prefixed with '!New Database State!:'. If the API call is only requesting data, then don't change the database state, but base your 'API Response' off what's in the database.
 """
     completion = ray.get(gpt3.remote(gpt3_input))
     completion = json.loads(completion)["text"]
 
-    future1 = gpt3.remote(f"{completion}\n\nAPI Response as valid json (as above, ignoring new database state): ")
-    future2 = gpt3.remote(f"{completion}\n\nThe value of 'New Database State' above (as json):")
-    response = json.loads(ray.get(future1).strip())["text"].strip()
+    # parsing "API Response" and "New Database State" with regex
+    future1 = re.search("(?<=!API Response!:).*(?=!New Database State!:)", completion, re.DOTALL)
```

Comment:
could we rename these variables from `futureX` to something more semantic

Reply:
Yes that would be a good idea. I will edit that. I just copied them from the code that was there before.